<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Osciloscopio</class>
 <widget class="QMainWindow" name="Osciloscopio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="labelLogo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>190</width>
       <height>190</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="grupoGrafica">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>341</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Configuración de la gráfica</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>V/División</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelvDivision">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>labelv</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>S/División</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelsDivision">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>labels</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="slidervDvision">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="slidersDvision">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="botonIniciarGrafica">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Iniciar</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="grupoSerial">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>201</width>
       <height>190</height>
      </rect>
     </property>
     <property name="title">
      <string>Configuración de la comunicación serial</string>
     </property>
     <widget class="QPushButton" name="botonIniciarSerial">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Verificar</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombrePuertoSerial">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Puerto</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
